### ISSUE TITLE
Drupal example provisioning fails

### ISSUE THREAD

**Issue by cwardgar** [geerlingguy/ansible-for-devops#56]
```
TASK [Install Drupal.] *********************************************************
fatal: [default]: FAILED! => {"changed": true, "cmd": ["drush", "si", "-y", "--site-name=Drupal Test", "--account-name=admin", "--account-pass=admin", "--db-url=mysql://drupaltest:1234@localhost/drupaltest"], "delta": "0:00:00.195786", "end": "2017-05-11 00:12:07.375858", "failed": true, "rc": 255, "start": "2017-05-11 00:12:07.180072", "stderr": "PHP Fatal error:  Class Symfony\\Component\\Console\\Style\\SymfonyStyle contains 4 abstract methods and must therefore be declared abstract or implement the remaining methods (Symfony\\Component\\Console\\Output\\OutputInterface::isQuiet, Symfony\\Component\\Console\\Output\\OutputInterface::isVerbose, Symfony\\Component\\Console\\Output\\OutputInterface::isVeryVerbose, ...) in /var/www/drupal-8.2.x-dev/vendor/symfony/console/Style/SymfonyStyle.php on line 33\n [error]  Drush command terminated abnormally due to an unrecoverable error.\nError: Class Symfony\\Component\\Console\\Style\\SymfonyStyle contains 4 abstract methods and must therefore be declared abstract or implement the remaining methods (Symfony\\Component\\Console\\Output\\OutputInterface::isQuiet, Symfony\\Component\\Console\\Output\\OutputInterface::isVerbose, Symfony\\Component\\Console\\Output\\OutputInterface::isVeryVerbose, ...) in /var/www/drupal-8.2.x-dev/vendor/symfony/console/Style/SymfonyStyle.php, line 33 ", "stderr_lines": ["PHP Fatal error:  Class Symfony\\Component\\Console\\Style\\SymfonyStyle contains 4 abstract methods and must therefore be declared abstract or implement the remaining methods (Symfony\\Component\\Console\\Output\\OutputInterface::isQuiet, Symfony\\Component\\Console\\Output\\OutputInterface::isVerbose, Symfony\\Component\\Console\\Output\\OutputInterface::isVeryVerbose, ...) in /var/www/drupal-8.2.x-dev/vendor/symfony/console/Style/SymfonyStyle.php on line 33", " [error]  Drush command terminated abnormally due to an unrecoverable error.", "Error: Class Symfony\\Component\\Console\\Style\\SymfonyStyle contains 4 abstract methods and must therefore be declared abstract or implement the remaining methods (Symfony\\Component\\Console\\Output\\OutputInterface::isQuiet, Symfony\\Component\\Console\\Output\\OutputInterface::isVerbose, Symfony\\Component\\Console\\Output\\OutputInterface::isVeryVerbose, ...) in /var/www/drupal-8.2.x-dev/vendor/symfony/console/Style/SymfonyStyle.php, line 33 "], "stdout": "", "stdout_lines": []}
```
I know nothing about Drupal so I'm kind of useless at debugging this. I can tell you that changing `drupal_core_version` to other tags in the `8.2.x` branch didn't help.

**Comment by rschmidtz** [comment]
I also got an error in the "Install Drupal" task, but this seems different:

`[error]  Command site-install needs a higher bootstrap level to run - you will need to invoke drush from a more functional Drupal environment to run this command.`

**Comment by scarroy** [comment]
I have the same error as rschmidtz shows. Not a huge deal since I think the bigger point was to illustrate how deploying a drupal site could be launched via ansible. Since I'm not proficient in drupal I can only google on the error message and hope for the best.

**Comment by geerlingguy** [comment]
Yeah, it's actually due to some changes that occurred in Drupal 8.3.x or Drush 8 in the past month. I'll get the examples updated as soon as I can!

**Comment by scarroy** [comment]
Thanks. So far one of the best Ansible books I've read!

**Comment by geerlingguy** [comment]
I'm testing this out locally, and hope to have an update this weekend to fix the problem. Thanks again for reporting/verifying!

**Comment by geerlingguy** [comment]
I'm also working on reproducing the error in a Travis build (https://travis-ci.org/geerlingguy/ansible-for-devops/jobs/236568399); that way it won't sneak back into a bad state again!

**Comment by geerlingguy** [comment]
Fixed! Thanks for reporting this; the problem was the `master` branch of Drush recently broke the way it worked with Drupal 8.2.x (which is an old release at this point anyways). So I switched the example to use Drush `8.x` (instead of `master`), and Drupal 8.3.x (instead of 8.2.x). I'll get an update pushed out to the book downloads soon (with credits in the errata to those who've commented on this issue). Thanks again!
